src/ContainerIntegration.jsx

```diff
@@ -89,18 +87,18 @@ const ContainerEnv = ({ containerEnv, imageEnv }) => {
     return <List isPlain>{result}</List>;
 };
 
-const ContainerIntegration = ({ container, containerDetail, localImages }) => {
-    if (containerDetail === null || localImages === null) {
+const ContainerIntegration = ({ container, localImages }) => {
+    if (localImages === null) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1324-20230712-150126-d59c3ba5-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)